src/big/big_decimal.cr

```diff
@@ -205,31 +207,77 @@ struct BigDecimal < Number
   # ```
   def div(other : BigDecimal, max_div_iterations = DEFAULT_MAX_DIV_ITERATIONS) : BigDecimal
```

Comment:
We should probably have a deprecated overload with named argument ˋmax_div_iterationsˋ and change the positional argument to ˋprecisionˋ. That should be a safe change.